sdk/storage/azure-storage-files-datalake/test/datalake_sas_test.cpp

```diff
@@ -119,27 +119,28 @@ namespace Azure { namespace Storage { namespace Test {
       {
       }
       std::string newFilename = RandomString();
-      auto newFileClient0 = directory2Client0.GetFileClient(newFilename);
-      newFileClient0.Create();
-      auto fileClient = Files::DataLake::FileClient(newFileClient0.GetDfsUri() + sas);
+      auto newDataLakeFileClient0 = directory2Client0.GetFileClient(newFilename);
+      newDataLakeFileClient0.Create();
```

Comment:
nah, we don't have to change this one, right?

This way, it's inconsistent with `fileClient` below.